processing_scripts/Workflow7_ImageJ_Jython_Analyst.py

```diff
@@ -0,0 +1,236 @@
+#-----------------------------------------------------------------------------
+#  Copyright (C) 2017 University of Dundee. All rights reserved.
+#
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  (at your option) any later version.
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+#  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+#------------------------------------------------------------------------------
+
+# This Jython script uses ImageJ to Subtract Background.
```

Comment:
Not sure, does it really subtract background ? I guess not, reading the ``IJ`` commands part below.